src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java

```diff
@@ -0,0 +1,44 @@
+package cs.ualberta.cmput402.tictactoe.scoreboard;
+
+import cs.ualberta.cmput402.tictactoe.board.Board.Player;
+
+import java.util.Hashtable;
+
+public class Scoreboard {
+
+    private Hashtable<Player, Integer> statistics;
+
+    public Scoreboard() {
+        statistics = new Hashtable<Player, Integer>();
+        statistics.put(Player.X, 0);
+        statistics.put(Player.O, 0);
+        statistics.put(Player.NONE, 0);
+    }
+
+    public void printScoreboard() {
```

Comment:
This method should be complemented to be explicitly according to the requirements, i.e., it must display the number of games won, lost and tied for each player.

Reply:
This change has been implemented.  Thank you for the feedback.